.github/workflows/causal_lm_cpp.yml

```diff
@@ -33,10 +33,9 @@ jobs:
           optimum-cli export openvino --trust-remote-code --weight-format fp16 --model openlm-research/open_llama_3b_v2 open_llama_3b_v2
           cmake -DCMAKE_BUILD_TYPE=Release -S ./text_generation/causal_lm/cpp/ -B ./build/
           cmake --build ./build/ --config Release -j
-      - name: convert_tokenizer and run
+      - name: greedy_causal_lm
         run: |
           source ./ov/setupvars.sh
-          convert_tokenizer ./open_llama_3b_v2/ --output ./open_llama_3b_v2/ --with-detokenizer
```

Comment:
looks like newer optimum version is required 
![image](https://github.com/openvinotoolkit/openvino.genai/assets/2566854/8f4b42c9-14f4-43c5-8d21-ceb26fc38fd5)